src/qforte/utils/qft.py

```diff
@@ -3,32 +3,34 @@
 
 
 
-def qft_circuit(n, direct):
+def qft_circuit(na, nb, direct):
```

Comment:
Can you please make sure that the function is well-behaved if `direct` is ill specified (e.g. due to a typo?). The right thing to do would be to raise an exception.

Reply:
Sure, I have added an exception here, thanks for the advice!